cmd/podmanV2/root.go

```diff
@@ -83,17 +134,62 @@ func syslogHook() {
+	}
+	flags.StringSliceVar(&cfg.Engine.ConmonPath, "conmon", []string{}, "Path of the conmon binary")
+	flags.StringVar(&cfg.Engine.NetworkCmdPath, "network-cmd-path", cfg.Engine.NetworkCmdPath, "Path to the command for configuring the network")
+	flags.StringVar(&cfg.Network.NetworkConfigDir, "cni-config-dir", cfg.Network.NetworkConfigDir, "Path of the configuration directory for CNI networks")
+	flags.StringVar(&cfg.Containers.DefaultMountsFile, "default-mounts-file", cfg.Containers.DefaultMountsFile, "Path to default mounts file")
+	if err := flags.MarkHidden("default-mounts-file"); err != nil {
+		logrus.WithError(err).Warn("unable to mark default-mounts-file flag as hidden")
+	}
+	flags.StringVar(&cfg.Engine.EventsLogger, "events-backend", cfg.Engine.EventsLogger, `Events backend to use ("file"|"journald"|"none")`)
+	flags.StringSliceVar(&cfg.Engine.HooksDir, "hooks-dir", cfg.Engine.HooksDir, "Set the OCI hooks directory path (may be set multiple times)")
+	flags.IntVar(&opts.MaxWorks, "max-workers", 0, "The maximum number of workers for parallel operations")
+	if err := flags.MarkHidden("max-workers"); err != nil {
+		logrus.WithError(err).Warn("unable to mark max-workers flag as hidden")
+	}
+	flags.StringVar(&cfg.Engine.Namespace, "namespace", cfg.Engine.Namespace, "Set the libpod namespace, used to create separate views of the containers and pods on the system")
+	flags.StringVar(&cfg.Engine.StaticDir, "root", "", "Path to the root directory in which data, including images, is stored")
+	flags.StringVar(&opts.Runroot, "runroot", "", "Path to the 'run directory' where all state information is stored")
+	flags.StringSliceVar(&opts.Engine.RuntimePath, "runtime", []string{}, "Path to the OCI-compatible binary used to run containers, default is /usr/bin/runc")
+	// -s is deprecated due to conflict with -s on subcommands
+	flags.StringVar(&opts.StorageDriver, "storage-driver", "", "Select which storage driver is used to manage storage of images and containers (default is overlay)")
+	flags.StringArrayVar(&opts.StorageOpts, "storage-opt", []string{}, "Used to pass an option to the storage driver")
+
+	flags.StringVar(&opts.Engine.TmpDir, "tmpdir", "", "Path to the tmp directory for libpod state content.\n\nNote: use the environment variable 'TMPDIR' to change the temporary storage location for container images, '/var/tmp'.\n")
+	flags.BoolVar(&opts.Trace, "trace", false, "Enable opentracing output (default false)")
+	if err := flags.MarkHidden("trace"); err != nil {
+		logrus.WithError(err).Warn("unable to mark trace flag as hidden")
+	}
+
+	// Override default --help information of `--help` global flag
+	var dummyHelp bool
+	flags.BoolVar(&dummyHelp, "help", false, "Help for podman")
+	flags.StringVar(&logLevel, "log-level", logLevel, fmt.Sprintf("Log messages above specified level (%s)", logLevels.String()))
+	flags.BoolVar(&useSyslog, "syslog", false, "Output logging information to syslog as well as the console (default false)")
```

Comment:
We should probably disable this for remote lie we do for max-workers and others

Reply:
I'll add this to rootless PR.